core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -412,18 +412,15 @@ $(document).ready(function () {
                     ],
                     "columnDefs": [
                         {
-                            "targets": [ 15 ],
-                            "visible": logsExist(response)
-                        },
-                        {
                             "targets": [ 16 ],
                             "visible": getThreadDumpEnabled()
                         }
                     ],
                     "order": [[0, "asc"]]
                 };
     
-                $(selector).DataTable(conf);
+                var dt = $(selector).DataTable(conf);
```

Comment:
You know better than I how to implement this. Does the same logic apply to column 16's init above?